luigi/worker.py

```diff
@@ -910,7 +914,8 @@ def _create_task_process(self, task):
         return TaskProcess(
             task, self._id, self._task_result_queue, reporter,
             use_multiprocessing=bool(self.worker_processes > 1),
-            worker_timeout=self._config.timeout
+            worker_timeout=self._config.timeout,
+            check_unfulfilled_deps=self._config.check_unfulfilled_deps
```

Comment:
Can you add a trailing comma here? Next line addition will look nicer in the diffs. :)